content/rs/concepts/high-availability/clustering.md

```diff
@@ -31,18 +31,15 @@ In clustering, these terms are commonly used:
 - Slot or Hash Slot - The result of the hash calculation.
 - Shard - Redis process that is part of the Redis clustered database.
 
-## When to use sharding
+## When to use clustering
```

Comment:
I think the right term here is actually sharding.
Sharding the data is the action of splitting the data into different Shards.

Also true for lines 36-37.


@AlonMagrafta ?

Reply:
@bmansheim I know we discussed it and we assumed the term Clustering (the page title) was the one to use. It is even defined in the first paragraph of this page.
But, looking on https://redis.io/topics/cluster-tutorial I'm starting to think @orshwartzredislabs might be right.
Maybe we need to look at this as sharding (not clustering) and the operation of dividing the data to shards. And clustering to have multiple nodes per cluster.
WDYT?